src/services/logger.ts

```diff
@@ -0,0 +1,13 @@
+import * as winston from "winston";
+
+export const logger = winston.createLogger({
+    level: 'info',
```

Comment:
the level shouldn't be configured here by default, it should be decided by the user:

logger.info(), logger.debug(), logger.log(), etc.